docs/rules/file-extension-in-import.md

```diff
@@ -11,9 +11,9 @@ import foo from "./path/to/a/file" // maybe it's resolved to 'file.js' or 'file.
 export * from "./path/to/a/file"
 ```
 
-However, [--experimental-modules](https://medium.com/@nodejs/announcing-a-new-experimental-modules-1be8d2d6c2ff) has declared to drop the file extension omition.
+However, when using [`--experimental-modules`](https://medium.com/@nodejs/announcing-a-new-experimental-modules-1be8d2d6c2ff), file extensions are required.
```

Comment:
esm has been stable in node.js, so we may not need to mention the flag?

Reply:
Agreed, I think this entire line can be removed
```suggestion
```